solr/core/src/java/org/apache/solr/core/SolrCore.java

```diff
@@ -805,6 +792,24 @@ public SolrCore reload(ConfigSet coreConfig) throws IOException {
     }
   }
 
+  /**
+   * Clones the current core for core reload, with the provided CoreDescriptor and ConfigSet.
+   *
+   * @return the cloned core to be used for SolrCore#load
```

Comment:
Not for SolrCore#reload but SolrCore#reload -- you could actually link to the method instead BTW